package apps.bi;import "collector.proto";message DataAuthor {	enum Type {		TOPIC = 0;		COMMENT = 1;	}		required Type type = 1;	required string uri = 2;	required string name = 3;}message DataComment {	required string uri = 1;	required DataAuthor author = 2;	required float time = 3;					/* time when the comment was posted in UNIX format*/	required string text = 4;					/* comment text without html*/	required string refs = 5;					/* reference inside the comment*/	required uint32 pos_raiting = 6;			/* number of comment positive votes*/	required uint32 neg_raiting = 7;			/* number of comment negative votes*/}message JobHubRange {	required collector.core.JobMeta meta = 1;	optional uint32 f = 2;						/* (from) start page*/	optional uint32 t = 3;						/* (to) end page*/	repeated uint32 sample = 4;					/* enumerated peges*/}message JobHubUri {	required collector.core.JobMeta meta = 1;	required string uri = 2;}message JobRawHub {	required collector.core.JobMeta meta = 1;	required string html = 2;}message JobRawTopic {	required collector.core.JobMeta meta = 1;		required string uri = 2;	required string title = 3;	required float post_time = 4;				/* time when a topic was posted in UNIX format*/	required int32 views = 5;					/* number of page views*/	required int32 comments = 6;				/* number of comments*/	required string short_text = 7;				/* short text from topic's hub*/	required string image_uri = 8;				/* image of topic inside topic's hub*/		optional string html = 9;	optional string full_text = 10;				/* full text from topic*/	optional bytes image_data = 11;				/* image from hub 'as is'*/	optional collector.core.DataSR response = 12;	optional collector.core.DataRSR response_raw = 13;		repeated string tags = 14;	repeated DataAuthor authors = 15;	repeated DataComment comments_data = 16;}